src/core.py

```diff
@@ -603,7 +622,11 @@ def translate(self, var):
                 for f in dc.fields(TranslatedVarlistEntry) if hasattr(var, f.name)}
             new_name = var.name
         else:
-            fl_entry = self.from_CF(var.standard_name, var.modifier, var.dims.__len__())
+            has_scalar_coords = False
+            if len(var.scalar_coords)>0:
+                has_scalar_coords = True
```

Comment:
Consider replacing with:

```suggestion
            has_scalar_coords = bool(var.scalar_coords)
```